WalletWasabi.Tests/RegressionTests/BackendTests.cs

```diff
@@ -202,8 +206,10 @@ public async Task StatusRequestTestAsync()
 
 				await rpc.GenerateAsync(1);
 
-				response = await client.TorClient.SendAsync(HttpMethod.Get, request);
-				using (HttpContent content = response.Content)
+				response = await BackendHttpClient.SendAsync(HttpMethod.Get, requestUri);
+				Assert.NotNull(response);
+
+				using (HttpContent content = response!.Content)
```

Comment:
If there is a `using` on the `response` isn't that disposing the content automatically?

Reply:
Copied from Slack:

https://source.dot.net/#System.Net.Http/System/Net/Http/HttpResponseMessage.cs,225 - it behaves like this
1:34
In my opinion, one should dispose everything that is disposable. So if a class is disposable, just dispose it. Even if it means that you dispose something twice (which is not a problem since the disposable pattern is by its nature idempotent)

Reply:
I am not sure. This can be confusing to others. "Do I have to dispose the content too?" So better be safe than sorry according to this precedent they will falsely add the disposal everywhere.

Reply:
I'm not sure what do you mean by "better be safe than sorry". In this case, "safe" is to dispose everything. 

But I believe it's actually more correct to dispose what is disposable. You don't know when .NET will change their internal implementation. So you dispose, what you work with and not as it is internally implemented.

Reply:
Exactly. The safe is to dispose everything, except if we know it is unnecessary and don't do it there consistently. If we do it once, then all of the places we have to do it - but it wouldn't make sense as it is done by HttpResponseMessage as you pointed out.